README-template.md

```diff
@@ -1,94 +1,42 @@
-![Greenbone Logo](https://www.greenbone.net/wp-content/uploads/gb_logo_resilience_horizontal.png)
+<!--
+SPDX-FileCopyrightText: 2023 Greenbone AG
 
-# Project Name
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
 
-[![GitHub releases](https://img.shields.io/github/release/greenbone/PROJECT.svg)](https://github.com/greenbone/PROJECT/releases)
-[![PyPI release](https://img.shields.io/pypi/v/PROJECT.svg)](https://pypi.org/project/PROJECT/)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/greenbone/PROJECT/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/greenbone/PROJECT/?branch=master)
-[![code test coverage](https://codecov.io/gh/greenbone/PROJECT/branch/master/graph/badge.svg)](https://codecov.io/gh/greenbone/PROJECT)
-[![CircleCI](https://circleci.com/gh/greenbone/PROJECT/tree/master.svg?style=svg)](https://circleci.com/gh/greenbone/PROJECT/tree/master)
+![Greenbone Logo](https://www.greenbone.net/wp-content/uploads/Gb_New-logo_horizontal_head.png)
 
-TODO: Replace PROJECT with project references!
+# Greenbone Software Component
```

Comment:
I think this should be a placeholder? Can we make that more obvious?

Reply:
```suggestion
# Greenbone <SOFTWARE COMPONENT>
```

Maybe like this?

Reply:
Yes, looks good.